ccp/modules/datashield-compose.yml

```diff
@@ -0,0 +1,106 @@
+version: "3.7"
```

Comment:
This is only needed at the top level docker-compose

Reply:
This works with some versions of Docker, but not with all of them.